### ISSUE TITLE
Error: Modlist

### ISSUE THREAD

**Issue by Witkh13** [ThirdEyeSqueegee/Ro#4]
The instructions were followed, with a fresh install, & Wabbajack ran into several errors.  The errors are listed below.  The problem seems to be that it is unable to download Creation Club files.  Why is it trying to download Creation Club content?  Anniversary Edition is installed, so I am unsure the nature of this problem... I guess I could copy the files over to the Wabbajack download folder so that it sees them already there...

`00:15:52.971 [INFO] (Wabbajack.Installer.StandardInstaller) Next Step: Hashing Archives
00:15:52.971 [INFO] (Wabbajack.Installer.StandardInstaller) Looking for files to hash
00:15:53.002 [INFO] (Wabbajack.Installer.StandardInstaller) Getting archive sizes
00:15:53.018 [INFO] (Wabbajack.Installer.StandardInstaller) Linking archives to downloads
00:15:53.018 [INFO] (Wabbajack.Installer.StandardInstaller) Found 2234 total files, 2154 matching filesize
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccbgssse060-ba_dragonscale.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccbgssse060-ba_dragonscale.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccrmssse001-necrohouse.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccrmssse001-necrohouse.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccbgssse069-contest.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccbgssse069-contest.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccbgssse062-ba_dwarvenmail.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccbgssse062-ba_dwarvenmail.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccasvsse001-almsivi.esm (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccasvsse001-almsivi.esm)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccbgssse020-graycowl.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccbgssse020-graycowl.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccbgssse061-ba_dwarven.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccbgssse061-ba_dwarven.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccbgssse040-advobgobs.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccbgssse040-advobgobs.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_cceejsse001-hstead.esm (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\cceejsse001-hstead.esm)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccbgssse063-ba_ebony.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccbgssse063-ba_ebony.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccbgssse005-goldbrand.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccbgssse005-goldbrand.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccbgssse016-umbra.esm (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccbgssse016-umbra.esm)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccbgssse064-ba_elven.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccbgssse064-ba_elven.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccbgssse068-bloodfall.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccbgssse068-bloodfall.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccmtysse002-ve.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccmtysse002-ve.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccbgssse052-ba_iron.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccbgssse052-ba_iron.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_cceejsse004-hall.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\cceejsse004-hall.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccbgssse034-mntuni.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccbgssse034-mntuni.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccmtysse001-knightsofthenine.bsa (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccmtysse001-knightsofthenine.bsa)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccbgssse053-ba_leather.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccbgssse053-ba_leather.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccbgssse011-hrsarmrelvn.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccbgssse011-hrsarmrelvn.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccbgssse054-ba_orcish.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccbgssse054-ba_orcish.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccbgssse012-hrsarmrstl.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccbgssse012-hrsarmrstl.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccffbsse001-imperialdragon.bsa (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccffbsse001-imperialdragon.bsa)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccbgssse055-ba_orcishscaled.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccbgssse055-ba_orcishscaled.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccbgssse021-lordsmail.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccbgssse021-lordsmail.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccbgssse056-ba_silver.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccbgssse056-ba_silver.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_cceejsse002-tower.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\cceejsse002-tower.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccbgssse057-ba_stalhrim.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccbgssse057-ba_stalhrim.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccafdsse001-dwesanctuary.esm (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccafdsse001-dwesanctuary.esm)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccbgssse058-ba_steel.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccbgssse058-ba_steel.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccvsvsse003-necroarts.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccvsvsse003-necroarts.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccbgssse014-spellpack01.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccbgssse014-spellpack01.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccbgssse041-netchleather.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccbgssse041-netchleather.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccbgssse002-exoticarrows.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccbgssse002-exoticarrows.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccbgssse035-petnhound.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccbgssse035-petnhound.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccpewsse002-armsofchaos.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccpewsse002-armsofchaos.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccedhsse001-norjewel.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccedhsse001-norjewel.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_cckrtsse001_altar.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\cckrtsse001_altar.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccvsvsse002-pets.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccvsvsse002-pets.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_cceejsse005-cave.esm (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\cceejsse005-cave.esm)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccbgssse003-zombies.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccbgssse003-zombies.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccbgssse036-petbwolf.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccbgssse036-petbwolf.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccbgssse038-bowofshadows.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccbgssse038-bowofshadows.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccedhsse003-redguard.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccedhsse003-redguard.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccqdrsse002-firewood.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccqdrsse002-firewood.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccbgssse004-ruinsedge.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccbgssse004-ruinsedge.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccbgssse007-chrysamere.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccbgssse007-chrysamere.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccasvsse001-almsivi.bsa (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccasvsse001-almsivi.bsa)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccffbsse001-imperialdragon.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccffbsse001-imperialdragon.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccvsvsse001-winter.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccvsvsse001-winter.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccbgssse013-dawnfang.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccbgssse013-dawnfang.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_cceejsse003-hollow.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\cceejsse003-hollow.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccbgssse031-advcyrus.esm (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccbgssse031-advcyrus.esm)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccbgssse018-shadowrend.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccbgssse018-shadowrend.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccmtysse001-knightsofthenine.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccmtysse001-knightsofthenine.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccedhsse002-splkntset.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccedhsse002-splkntset.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccbgssse010-petdwarvenarmoredmudcrab.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccbgssse010-petdwarvenarmoredmudcrab.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccbgssse045-hasedoki.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccbgssse045-hasedoki.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccbgssse043-crosselv.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccbgssse043-crosselv.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccbgssse019-staffofsheogorath.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccbgssse019-staffofsheogorath.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccffbsse002-crossbowpack.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccffbsse002-crossbowpack.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccbgssse066-staves.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccbgssse066-staves.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccfsvsse001-backpacks.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccfsvsse001-backpacks.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccvsvsse004-beafarmer.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccvsvsse004-beafarmer.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccbgssse006-stendarshammer.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccbgssse006-stendarshammer.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccbgssse051-ba_daedricmail.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccbgssse051-ba_daedricmail.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_cccbhsse001-gaunt.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\cccbhsse001-gaunt.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccbgssse008-wraithguard.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccbgssse008-wraithguard.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccbgssse050-ba_daedric.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccbgssse050-ba_daedric.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccbgssse059-ba_dragonplate.esl (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccbgssse059-ba_dragonplate.esl)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_cctwbsse001-puzzledungeon.esm (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\cctwbsse001-puzzledungeon.esm)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Unable to download Data_ccbgssse067-daedinv.esm (GameFileSourceDownloader+State|SkyrimSpecialEdition|1.6.640.0|data\ccbgssse067-daedinv.esm)
00:15:53.161 [FATAL] (Wabbajack.Installer.StandardInstaller) Cannot continue, was unable to download one or more archives`

**Comment by Witkh13** [comment]
Update: copying the AE files into Ro's 'download' folder seems to have worked.  Still would be easier if that step was elucidated in the "Installation" directions, think do I.

In addition there is a new problem.  The installation was completed following the steps.  However, it now gives me the following error.  The overall point of the error is that there fails to be a  Skyrim Script Extender installed into the game.  That is humorous given the instructions of applying the modlist to a fresh install of the game, so of course I had yet to install anything, let alone skse64_loader.exe in the games own directory.

On a tangential note, the directions said to use MO that was installed with the modlist yet that fails to actually exist after Wabbajack said it was done with the installation.  I had to find the zip file that was downloaded & then pulled that into the installation files from the modlist.  Then I had to edit MO to find th actual game through GoG's game folder instead of looking for it in Steam.  That was easy to fix.  The problem is everything else that fails to make any sense to me.

The installation folder fails to show any mods at all, so it seems suspect to just copy it into the game files & pray something works.

`Error 2 ERROR_FILE_NOT_FOUND: Thesystemcannotfindthefilespecified. (0x2)
 . binary: 'D:\Program Files (x86)\GOG Galaxy\Games\Skyrim Anniversary Edition\skse64_loader.exe'
 . owner: (file not found)
 . rights: (file not found)
 . arguments: ''
 . cwd: 'D:\Program Files (x86)\GOG Galaxy\Games\Skyrim Anniversary Edition'
 . stdout: no, stderr: no, hooked: yes
 . MO elevated: yes
 . usvfs x86:ok x64:ok proxy_x86:ok proxy_x64:ok`
